docs/how-to-guides/feature-flags.md

```diff
@@ -71,7 +74,7 @@ if ( undefined ) { // Wepack has replaced `process.env.IS_GUTENBERG_PLUGIN` with
 
 ### Dead code elimination
 
-For production builds, webpack ['minifies'](https://en.wikipedia.org/wiki/Minification_(programming)) the code, removing as much unnecessary JavaScript as it can. 
+For production builds, webpack ['minifies'](<https://en.wikipedia.org/wiki/Minification_(programming)>) the code, removing as much unnecessary JavaScript as it can.
```

Comment:
Bad angle brackets in link here.

Reply:
I _think_ this is actually prettier fixing the URL? At any rate, parens in link URLs in markdown is confusing and it's a good idea to URL encode them.

Moving to https://github.com/WordPress/gutenberg/pull/61675